xml/System.Security.Cryptography/DSAOpenSsl.xml

```diff
@@ -103,9 +153,20 @@
         <Parameter Name="parameters" Type="System.Security.Cryptography.DSAParameters" />
       </Parameters>
       <Docs>
-        <param name="parameters">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="parameters">The key parameters to import.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.DSAOpenSsl" /> class and imports parameters from an existing key.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="parameters"/> is missing required fields.
+
+-or-
+
+<paramref name="parameters"/> has fields with inconsistent lengths for a valid key.
+        </exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">
+          <paramref name="parameters"/> does not represent a valid DSA key parameter set.</exception>
+        <altmember cref="M:System.Security.Cryptography.DSA.Create(System.Security.Cryptography.DSA.Create(System.Security.Cryptography.DSAParameters)" />
```

Comment:
Malformed DocId. There's a Create call inside the Create call. I removed all the similar instances in this file.